taskcluster/mozillavpn_taskgraph/transforms/beetmover.py

```diff
@@ -13,33 +13,78 @@
 @transforms.add
 def add_beetmover_worker_config(config, tasks):
     for task in tasks:
-        app_name = "vpn"
         worker_type = task["worker-type"]
         is_relpro = (
             config.params["level"] == "3"
             and config.params["tasks_for"] in task["run-on-tasks-for"]
         )
         bucket = "release" if is_relpro else "dep"
-        build_id = config.params["moz_build_date"]
+        moz_build_date = config.params["moz_build_date"]
```

Comment:
I don't have a strong preference, but just fyi `build_id` is a fairly standard terminology for this in the Gecko world.

Reply:
Got it